src/tile.cpp

```diff
@@ -874,9 +877,8 @@ void Tile::addThing(int32_t, Thing* thing)
 			} else {
 				const ItemType& oldType = Item::items[ground->getID()];
 
-				Item* oldGround = ground;
+				auto oldGround = ground;
```

Comment:
Could this be `const auto`
```suggestion
				const auto oldGround = ground;
```